Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM node:6
+FROM node:7
```

Comment:
Why this version bump? I was under the impression that all ovirt projects have always the current LTS version of Node.js as their base runtime - currently v6. I believe our next bump should come when Node.js 8 enters LTS phase in october of this year https://github.com/nodejs/LTS#lts-schedule1